db-tests/src/test/java/gov/nasa/jpl/aerie/database/DatabaseTestHelper.java

```diff
@@ -67,6 +67,27 @@ public void startDatabase() throws SQLException, IOException, InterruptedExcepti
       proc.destroy();
     }
 
+    // Grant usage privileges to all aerie for all schemas for tests
```

Comment:
For my own understanding, what goes wrong if we don't grant these privileges, and why is does this occur in tests, but not in production?

Reply:
If we don't grant USAGE privileges in tests, then the `aerie` user cannot see anything not inside the public schema, meaning that, for example, it does not have access to `hasura_functions` when testing against AerieMerlin (which is needed for the `AnchorDeletion` and `AnchorMerge` tests).

From what I can tell looking between `init-aerie.sh` and `DatabaseTestHelper`, the difference between test and production seems to be that the `postgres` user creates all the objects in the db in Test, while the `aerie` user creates them in production (meaning it would automatically have access to all objects).

Reply:
I see, thanks for explaining. I think rather than grant aerie additional permissions in tests, we should use the aerie user to create those tables - especially since that helps mirror production a bit better.

I think we can do that by splitting this call in two - so that we leave the GRANT PRIVILEGES call as is, but run the `init-aerie.sh` under the `aerie` user (by replacing `postgres:postgres` in the url with `aerie:aerie`):

https://github.com/NASA-AMMOS/aerie/blob/37e6b229a852a618c7d243abb5acae4079526871/db-tests/src/test/java/gov/nasa/jpl/aerie/database/DatabaseTestHelper.java#L57-L61

Reply:
Makes sense. Change has been done 👍